code_samples/authorization/get_decision.mdx

```diff
@@ -123,6 +207,69 @@ public class GetDecisions {
 <TabItem value="js" label="Javascript">
 
 ```javascript
+import { PlatformClient } from '@opentdf/sdk/platform';
+import { AuthProviders } from '@opentdf/sdk';
+import { create } from '@bufbuild/protobuf';
+import { GetDecisionsRequestSchema, DecisionRequestSchema, Entity_CategorySchema } from '@opentdf/sdk/platform';
+
+async function main() {
+  const platformEndpoint = 'http://localhost:8080';
+  
+  // Assume you have an existing access token
+  const accessToken = 'your-access-token-here';
+
+  // Create auth provider with existing token
+  const authProvider = await AuthProviders.accessTokenAuthProvider({
```

Comment:
AFAIK `accessTokenAuthProvider` does not exist. There is a `refreshAuthProvider` here 
https://github.com/opentdf/web-sdk/blob/5c37f269ee4f74ce3e69e8474b70c1acc0337c8b/lib/src/auth/providers.ts#L86